website/docs/developer/deployment-guide/deployment-with-cumulus.md

```diff
@@ -351,43 +352,44 @@ the ingest workflow.
 
 ```json
 "CopyToGlacier":{
-   "Parameters":{
-      "cma":{
-         "event.$":"$",
-         "task_config":{
-            "multipart_chunksize_mb": "{$.meta.collection.meta.multipart_chunksize_mb"},
-            "excludeFileTypes": "{$.meta.collection.meta.excludeFileTypes}",
-            "providerId": "{$.meta.provider.id}",
-            "executionId": "{$.cumulus_meta.execution_name}",
-            "collectionShortname": "{$.meta.collection.name}",
-            "collectionVersion": "{$.meta.collection.version}",
-            "orcaDefaultBucketOverride": "{$.meta.collection.meta.orcaDefaultBucketOverride}"
-            }
-         }
-      }
-   },
-   "Type":"Task",
-   "Resource":"module.orca.orca_lambda_copy_to_glacier_arn",
-   "Catch":[
-      {
-         "ErrorEquals":[
-            "States.ALL"
-         ],
-         "ResultPath":"$.exception",
-         "Next":"WorkflowFailed"
-      }
-   ],
-   "Retry": [
-      {
-        "ErrorEquals": [
-           "States.ALL"
-        ],
-        "IntervalSeconds": 2,
-        "MaxAttempts": 3,
-        "BackoffRate": 2
+  "Parameters":{
+    "cma":{
+      "event.$":"$",
+      "task_config": {
+        "excludeFileTypes": "{$.meta.collection.meta.excludeFileTypes}",
+        "multipart_chunksize_mb": "{$.meta.collection.meta.multipart_chunksize_mb}",
```

Comment:
I am wondering if we shouldn't change the setting here from `"{$.meta.collection.meta.multipart_chunksize_mb}"` to `"{$.meta.collection.meta.s3MultipartChunksizeMb}"` so users only have to set this value in one place instead of two. Ideally we would also change from `multipart_chunksize_mb` to `s3MultipartChunksizeMb` as well.

Reply:
https://bugs.earthdata.nasa.gov/browse/ORCA-338